back-end/Recognition/src/utils.py

```diff
@@ -0,0 +1,49 @@
+from google.cloud import vision
+
+def process_image(image_path):
+    """
+    Processes an image file using the Google Vision API to perform text detection and label detection.
+
+    Args:
+        image_path (str): The file path of the image to process.
+
+    Returns:
+        tuple: A tuple containing the text annotations and label words as dictionaries.
+
+    Raises:
+        FileNotFoundError: If the image file cannot be found.
```

Comment:
you can delete those after the changes we made up there becouse theres no need to get file path any more